.github/workflows/scalability.yml

```diff
@@ -1,28 +1,35 @@
-name: Scalability Workflow
-# This workflow is triggered on pushes to the repository.
+name: Scalability Workflow updated
+
 on:
   push:
-    tags: 
-      - "*-beta.*"
-      
```

Comment:
I assume this change will be reverted too? Or are you suggesting we run scalability more often?

Reply:
Yes, correct, I'll revert to that before pushing to master.